Marlin/temperature.cpp

```diff
@@ -689,6 +689,7 @@ void manage_heater() {
   if (!temp_meas_ready) return;
 
   updateTemperaturesFromRawValues();
+  Regulator__Update();
```

Comment:
I think this should be added under REGULATOR_CHECK_INTERVAL. We don't great time resolution on it. I think REGULATOR_CHECK_INTERVAL should be reduced to ~1s (from 2.5s). @pizzyflavin thoughts?